sql/moz-fx-data-marketing-prod/ga_derived/downloads_with_attribution_v1/schema.yaml

```
@@ -129,26 +129,25 @@ fields:
   mode: NULLABLE
   name: count_dltoken_duplicates
   type: INTEGER
-- description: "This column represents if there was at least one other download completed during the same\
-    \ session as this download. "
+- description: 'This column represents if there was at least one other download completed
```

For the column descriptions you shouldn't need the quotes:
```suggestion
- description: Indicates if at least one more download was completed during the
    same session of the current download.
```

---

The quotes and formatting is odd, running `schema update` seems to add them.

---

Ah, pobably a recent change. I normally don't use them, like [in the aggregates](https://github.com/mozilla/bigquery-etl/blob/b5facd7b69fa9c8ec1e735e7e81ee27651abb07b/sql/moz-fx-data-shared-prod/telemetry_derived/active_users_aggregates_v1/schema.yaml#L86-L87), and I find the query much easier to read.

---

I removed them all the reran schema update, the quotes were not added back.